src/vm/customattribute.cpp

```diff
@@ -166,8 +166,15 @@ CustomAttributeManagedValues Attribute::GetManagedCaValue(CaValue* pCaVal)
         BOOL bAllBlittableCa = arrayType != SERIALIZATION_TYPE_ENUM;
 
         if (length == (ULONG)-1)
+        {
+            GCPROTECT_BEGIN(gc)
+            {   
+                if (arrayType == SERIALIZATION_TYPE_ENUM)
```

Comment:
ok, let me see if I can quickly change this via git online... so, you're suggesting to start `GCPROTECT_BEGIN(gc) {` after `CustomAttributeManagedValues gc;` and to close `} GCPROTECT_END();` before `return gc;` corect? Thanks